src/The_Cartridge_Header.md

```diff
@@ -1,65 +1,71 @@
 # The Cartridge Header
 
-An internal information area is located at $0100-014F in each cartridge.
-It contains the following values:
+Each cartridge contains a header, located at the address range `$0100`-`$014F`.
+The cartridge header provides the following information about the game itself and the hardware it expects to run on:
 
-## 0100-0103 - Entry Point
+## 0100-0103 – Entry Point
 
-After displaying the Nintendo Logo, the built-in boot procedure jumps to
-this address ($0100), which should then jump to the actual main program
-in the cartridge. Usually this 4 byte area contains a `nop` instruction,
-followed by a `jp $0150` instruction. But not always.
+After displaying the Nintendo logo, the built-in [boot ROM](<#Power-Up Sequence>) jumps to
+to address `$0100`, which should then jump to the actual main program
+in the cartridge.
+Most commercial games fill this 4-byte area with a [`nop` instruction](https://rgbds.gbdev.io/docs/v0.5.2/gbz80.7/#NOP) followed by a [`jp $0150`](https://rgbds.gbdev.io/docs/v0.5.2/gbz80.7/#JP_n16).
 
-## 0104-0133 - Nintendo Logo
+## 0104-0133 – Nintendo Logo
+
+These bytes contain a bitmap that is displayed when the Game Boy is powered on.
+They must match the following (hexadecimal) dump, otherwise [the boot ROM](<#Power-Up Sequence>) won't allow the game to run:
 
-These bytes define the bitmap of the Nintendo logo that is displayed
-when the Game Boy gets turned on. The hex dump of this bitmap is:
 ```
- CE ED 66 66 CC 0D 00 0B 03 73 00 83 00 0C 00 0D
- 00 08 11 1F 88 89 00 0E DC CC 6E E6 DD DD D9 99
- BB BB 67 63 6E 0E EC CC DD DC 99 9F BB B9 33 3E
+CE ED 66 66 CC 0D 00 0B 03 73 00 83 00 0C 00 0D
+00 08 11 1F 88 89 00 0E DC CC 6E E6 DD DD D9 99
+BB BB 67 63 6E 0E EC CC DD DC 99 9F BB B9 33 3E
 ```
-The Game Boy's boot procedure verifies the content of this bitmap
-(after it has displayed it), and LOCKS ITSELF UP if these bytes are
-incorrect. A CGB verifies only the first half ($18 bytes of) the bitmap, but
-others (for example a Game Boy pocket) verify all $30 bytes, as does the
-Game Boy Advance.
 
```

Comment:
Adding an explanation of how the thing is encoded might be a good idea?

```suggestion

The way pixels are encoded is as follows: ([more visual aid](https://github.com/ISSOtm/gb-bootroms/blob/2dce25910043ce2ad1d1d3691436f2c7aabbda00/src/dmg.asm#L259-L269))

- $0104–011B encode the top half of the logo, $011C–0133 encode the bottom half of it.
- For each half, each nibble encodes 4 pixels (the MSB corresponds to the leftmost pixel, the LSB to the rightmost); a pixel is lit if its bit is set.
- The 4-pixel "groups" are laid out top to bottom, left to right.
- And finally, the monochrome models upscale the entire thing by a factor of 2 (leading to somewhat chunky pixels).

```

Reply:
Is the logo encoded in a different manner than regular tiles/sprites? I don't remember off the top of my head, but basically what I am asking is that could the explanation be replaced with a link to the pertinent section of the PPU documentation?

Reply:
Yes, it is encoded differently. https://github.com/ISSOtm/gb-bootroms/blob/2dce25910043ce2ad1d1d3691436f2c7aabbda00/src/dmg.asm#L221